troposphere/static/js/components/Master.jsx

```diff
@@ -55,6 +57,15 @@ export default React.createClass({
         // The code below is only relevant to logged in users
         if (!context.hasLoggedInUser()) return;
 
+        if (Raven && window.SENTRY_DSN){
+            if (! Raven.isSetup()) {
```

Comment:
This is fine. I think an improvement would be removing the if statement, find out where in the app this should be setup (the bootstrapper) and make sure it's done there.